content/people/_index.md

```diff
@@ -72,10 +72,13 @@ preface = "For people participaing in and interacting with the consortium, there
 	name = "Alex Wolf"
 	url = "https://falexwolf.de/"
 
-
 [[teams.members]]
 	name = "Nir Yosef"
 	url = "https://yoseflab.github.io/"
+	
+[[teams.members]]
+	name = "Francesca Finotello"
+	url = "https://computationalbiomedicinegroup.github.io/"
```

Comment:
should we go for alphabetical order also for the advisory board?

Reply:
I was wondering about order for both this and core-devs/ contributors.

Let's push to an issue?

Reply:
sounds good! Core developers seem to be already (kinda?) alphabetical

Reply:
Just to clarify, in the names lists, I currently put items in the alphabetical order (by the last name).